.travis.yml

```diff
@@ -8,6 +8,14 @@ os:
 #  - osx add osx back when travis fixes their mac os scaling problems.
   - linux
 
+addons:
+  apt:
+    sources:
+    # add PPAs with more up-to-date toolchains
+    - llvm-toolchain-precise-3.8
+    packages:
+    - clang-format-3.8
```

Comment:
if it's possible to move the cppcheck install here, we could lose the sudo and move to sandboxed builds.

Reply:
I'll try that in a separate PR